gitlab_users

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+from lib import get_gitlab_instance
+
+import sys
+
+SHOW_CONFIG = len(sys.argv) >= 2 and sys.argv[1] == 'config'
+
+if SHOW_CONFIG:
+    print('graph_title GitLab users')
+    print('graph_vlabel users')
+    print('graph_args -l 0')
+    print('graph_category gitlab')
```

Comment:
That's caused in the way munin works. Munin first will run the plugin with the ```config``` parameter to set up the graph parameters. It's *not* expected to retreive any values at tihs moment (config will executed on munin-node start/restart).

Values are only expected while the normal (mostly cron based) run, called without any parameters.

On each cron run, munin does *not* expect the config